index.html

```diff
@@ -0,0 +1,354 @@
+            sign-ups required.
+          </p>
+          <ul class="category">
+            <li>html</li>
+            <li>css</li>
+            <li>javaScript</li>
+          </ul>
+          <button class="show">See Project</button>
+        </div>
+      </div>
+
+      <!-- Second Card-->
+      <div class="card-body">
+        <div class="card-img">
+          <img
+            src="images/image2.png"
+            alt="Project 2"
+            width="295px"
+            height="220px"
+          />
+        </div>
+
+        <h2 class="card-header">Multi-Post <br />Stories</h2>
+        <ul class="card-sub">
+          <li class="remove">CANOPY</li>
+          <li>Back End Dev</li>
+          <li>2015</li>
+        </ul>
+        <div class="card-content">
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="category">
+            <li>html</li>
+            <li>css</li>
+            <li>javaScript</li>
+          </ul>
+          <button class="show">See Project</button>
+        </div>
+      </div>
+
+      <!-- Thiird Card-->
+      <div class="card-body">
+        <div class="card-img">
+          <img
+            src="images/image3.png"
+            alt="Project 3"
+            width="295px"
+            height="220px"
+          />
+        </div>
+
+        <h2 class="card-header">Tonic</h2>
+        <ul class="card-sub">
+          <li class="remove">CANOPY</li>
+          <li>Back End Dev</li>
+          <li>2015</li>
+        </ul>
+        <div class="card-content">
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="category">
+            <li>html</li>
+            <li>css</li>
+            <li>javaScript</li>
+          </ul>
+          <button class="show">See Project</button>
+        </div>
+      </div>
+
+      <!-- Fourth Card-->
+      <div class="card-body">
+        <div class="card-img">
+          <img
+            src="images/image4.png"
+            alt="Project 4"
+            width="295px"
+            height="220px"
+          />
+        </div>
+
+        <h2 class="card-header">Multi-Post <br />Stories</h2>
+        <ul class="card-sub">
+          <li class="remove">CANOPY</li>
+          <li>Back End Dev</li>
+          <li>2015</li>
+        </ul>
+        <div class="card-content">
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="category">
+            <li>html</li>
+            <li>css</li>
+            <li>javaScript</li>
+          </ul>
+          <button class="show">See Project</button>
+        </div>
+      </div>
+    </div>
+
+    <section class="banner">
+      <h1 class="banner-title">About Myself</h1>
```

Comment:
Kindly update this element to use another `<hx>` element apart from `<h1>`🙏. This is for semantic reasons, as a good webpage is typically supposed to have one `<h1>` tag which represents the main title